api/client/formatter/custom.go

```diff
@@ -6,33 +6,37 @@ import (
 	"strings"
 	"time"
 
+	"github.com/docker/engine-api/types"
+	"github.com/docker/go-units"
 	"github.com/hyperhq/hypercli/api"
 	"github.com/hyperhq/hypercli/pkg/stringid"
 	"github.com/hyperhq/hypercli/pkg/stringutils"
-	"github.com/docker/engine-api/types"
-	"github.com/docker/go-units"
 )
 
 const (
 	tableKey = "table"
 	fipLabel = "sh.hyper.fip"
 
-	containerIDHeader  = "CONTAINER ID"
-	imageHeader        = "IMAGE"
-	namesHeader        = "NAMES"
-	commandHeader      = "COMMAND"
-	createdSinceHeader = "CREATED"
-	createdAtHeader    = "CREATED AT"
-	runningForHeader   = "CREATED"
-	statusHeader       = "STATUS"
-	portsHeader        = "PORTS"
-	sizeHeader         = "SIZE"
-	labelsHeader       = "LABELS"
-	imageIDHeader      = "IMAGE ID"
-	repositoryHeader   = "REPOSITORY"
-	tagHeader          = "TAG"
-	digestHeader       = "DIGEST"
-	fipHeader          = "PUBLIC IP"
+	containerIDHeader     = "CONTAINER ID"
+	imageHeader           = "IMAGE"
+	namesHeader           = "NAMES"
+	commandHeader         = "COMMAND"
+	createdSinceHeader    = "CREATED"
+	createdAtHeader       = "CREATED AT"
+	runningForHeader      = "CREATED"
+	statusHeader          = "STATUS"
+	portsHeader           = "PORTS"
+	sizeHeader            = "SIZE"
+	labelsHeader          = "LABELS"
+	imageIDHeader         = "IMAGE ID"
+	repositoryHeader      = "REPOSITORY"
+	tagHeader             = "TAG"
+	digestHeader          = "DIGEST"
+	fipHeader             = "PUBLIC IP"
+	volumeNameHeader      = "NAME"
+	volumeSizeHeader      = "SIZE"
+	volumeDriverHeader    = "DRIVER"
```

Comment:
All volumes are created with `hyper` driver. Not sure if we should let users filter drivers.